pkg/controller/operands/kubevirt.go

```diff
@@ -115,13 +106,15 @@ func NewKubeVirt(hc *hcov1beta1.HyperConverged, opts ...string) (*kubevirtv1.Kub
 		Workloads:         hcoConfig2KvConfig(hc.Spec.Workloads),
 	}
 
-	fgs := hc.Spec.FeatureGates.GetFeatureGateList(managedKvFeatureGates)
-	if len(fgs) > 0 {
-		if spec.Configuration.DeveloperConfiguration == nil {
-			spec.Configuration.DeveloperConfiguration = &kubevirtv1.DeveloperConfiguration{}
-		}
+	if hc.Spec.FeatureGates != nil {
+		fgs := hc.Spec.FeatureGates.GetFeatureGateList()
```

Comment:
Currentlly there are only KubeVirt gates, and they are collected by name. Each operand handler will take what relevant to its operand.

Reply:
The point I'm trying to make is, that when we'll add feature gates to another operand, it may break this code that assumes that "there are only KubeVirt gates". The assumption that the developer who'll add the next feature gate will know/remember to come back to `operands/kubevirt.go` to fix this is optimistic, but not necessarily holds.

Reply:
Let me try to explain again: this code only run in kv handler. Other handlers will have their own implementation. I don't think that code assumes that the fgs are only for kv.

Reply:
If, say, CDI adds a feature gate, won't it go under `HCO.Spec.FeatureGates.<CDI-Feature-Name>` and then returned from `GetFeaturedGateList()`, and transferred to virt-handler?